programs/disks/ICommand.cpp

```diff
@@ -30,17 +30,18 @@ void ICommand::addOptions(ProgramOptionsDescription & options_description)
     options_description.add(*command_option_description);
 }
 
-String ICommand::fullPathWithValidate(const DiskPtr & disk, const String & path)
+String ICommand::validatePathAndGetAsRelative(const String & path)
```

Comment:
Please check me here. Previous check relied on `fs::canonical` but this function require `path` to exists on local fs. It's not correct, because disk is abstraction which can be implemented independently of local filesystem. I've tried to test it with DiskMemory but it appeared to be complete unusable garbage and I've removed it.

Reply:
New way look Ok.